app/views/public_recipes/index.html.erb

```diff
@@ -0,0 +1,20 @@
+<h1 class= 'main-text align-text-center'>Public Recipes<h1>
+<% @recipes.each do |recipe, index|%>
+  <div class='card'>
+    <div class="flex justify-content-sa font-size-pr">
+      <div>
+        <p class='has-text-weight-bold is-size-4'>
+          <%= link_to recipe.name, recipe_path(id:recipe.id), class: 'text' %>
+        </p>
+        <p class='has-text-weight-semibold is-size-5'>By: <%= recipe.user.name%></p>
+      </div>
+      <% @recipe_foods = RecipeFood.joins(:food, :recipe).where(food: { user: recipe.user_id }, recipe: { user: recipe.user_id }) %>
+      <% @total_items = @recipe_foods.where(recipe_id: recipe.id).count %>
+      <% @total_cost = @recipe_foods.where(recipe_id: recipe.id).sum('quantity*price') %>
+      <div>
+        <p>Total food items: <%= @total_items%></p>
+        <p>Total price: $<%= @total_cost %></p>
+      </div>
```

Comment:
- [ OPTIONAL] SAme here 👏🏻  Just make sure the index page for Public recipe list matches the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/public_recipe_index.png?raw=true) I will be better to add unities: `grams` as the Quantity's unit as you added `$` as a price's unit price 😉 👍🏻 

- Your Design:

![public](https://user-images.githubusercontent.com/79058364/175202620-d8afbe3e-094c-49bb-8fc5-5c58f6765d8b.png)


- Original Design:

![publiucgod](https://user-images.githubusercontent.com/79058364/175203391-b911b7e4-4205-4dfd-bb73-6a20fafe6422.png)

Reply:
Hi @BenMukebo I have checked the app and it is showing the correct unit of measure, as you can see here:
![image](https://user-images.githubusercontent.com/10048317/175318664-04ff60ab-9de3-4442-883a-aeb05e92cafd.png)
And the source code looks right:
![image](https://user-images.githubusercontent.com/10048317/175319050-e606ce9f-e2d5-4d97-a6de-abf6c4272e24.png)
In your food's image the unit of measure contains numbers, so that is the reason.